README.md

```diff
@@ -25,7 +25,7 @@ analysis in order to understand your software's performance and behavior. You
 can export and analyze them using [Prometheus], [Jaeger], and other
 observability tools.
 
-*Compiler support: [requires `rustc` 1.60+][msrv]*
+*Compiler support: [requires `rustc` 1.64+][msrv]*
```

Comment:
Are there any implications of incrementing MSRV? Specifically, I'm concerned about the impact on applications that utilize our SDKs and are currently on a lower toolchain. Upgrading would mean enforcing them to use a higher compiler version. There could be features deprecated/removed in higher versions, and the applications still be using them.

Probably, I am thinking this with a C++ mindset There, we often strive to support the earliest versions across various compilers. For example, for C++11 support, we make sure that we can build successfully with the lowest supported GCC version, such as gcc-4.8. It seems that upgrades are more frequent in the Rust ecosystem. :)

Reply:
I think it's ok to update as tokio updated to 1.63+ in https://github.com/tokio-rs/tokio/pull/5887, this project is less foundational to the rust ecosystem than they are.

Reply:
Yes, that makes sense.